pom.xml

```diff
@@ -819,6 +836,16 @@
         <module>querydsl-maven-plugin</module>
       </modules>
     </profile>
+    
+    <profile>
+      <id>osgi</id>
```

Comment:
What is the purpose of this `osgi` module?

Reply:
I wanted to select out these integration tests for OSGI.